tests/runtime/servicers/test_training_management_servicer.py

```diff
@@ -258,7 +258,7 @@ def test_training_status_incorrect_id(training_management_servicer):
 
     assert context.value.status_code == grpc.StatusCode.NOT_FOUND
     assert (
-        f"Unknown training_id: some_random_id" in context.value.message
+        f"Unknown background_id: some_random_id" in context.value.message
```

Comment:
I think these need to be reverted. If they are needed to get the tests to pass, I think the source error message needs to be reverted!